internal/credential/vault/ssh_certificate_credential_library.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 const (
-	KeyTypeEcdsa   = "ecdsa"
+	KeyTypeEcdsa   = "ec"
```

Comment:
this (as well as references in postgres) were updated from `ecdsa` to `ec` to match vault

Reply:
This should stay as `ecdsa`, see the comment on the migration.